Gateway/Data/AddressAdapterInterface.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
```

Comment:
Please include copyright doc block for the files.
```suggestion
/**
 * Copyright © Magento, Inc. All rights reserved.
 * See COPYING.txt for license details.
 */
```